src/Feeds/Target/ExtendedDateTimeFormat.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Drupal\controlled_access_terms\Feeds\Target;
+
+use Drupal\controlled_access_terms\EDTFUtils;
+use Drupal\feeds\Exception\TargetValidationException;
+use Drupal\feeds\Feeds\Target\StringTarget;
+
+
+/**
+ * Defines an edtf field mapper.
+ *
+ * @FeedsTarget(
+ *   id = "edtf_feeds_target",
+ *   field_types = {"edtf"}
+ * )
+ */
+
+class ExtendedDateTimeFormat extends StringTarget {
+  protected $settings;
+  /**
+   * {@inheritdoc}
+   */
+  public function __construct(array $configuration, $plugin_id, array $plugin_definition) {
+    $this->targetDefinition = $configuration['target_definition'];
+    $this->settings = $this->targetDefinition->getFieldDefinition()->getSettings();
+    parent::__construct($configuration, $plugin_id, $plugin_definition);
+  }
+
```

Comment:
We need to provide a `buildConfigurationForm` function to allow site administrators to update the various settings used in the `prepareValue` function ('intervals','sets', and 'strict_dates').